site/src/helpers/planner.ts

```diff
@@ -133,13 +135,13 @@ export const loadRoadmap = async (
   // if logged in
   if (cookies.user !== undefined) {
     // get data from account
-    const request = await axios.get<MongoRoadmap>('/api/roadmap/get', { params: { id: cookies.user.id } });
+    const request = await trpc.roadmaps.get.query({ userID: cookies.user.id });
     // if a roadmap is found
-    if (request.data.roadmap !== undefined) {
-      roadmap = request.data.roadmap;
+    if (request?.roadmap) {
+      roadmap = request.roadmap;
     }
-    if (request.data.coursebag !== undefined) {
-      coursebagStrings = request.data.coursebag;
+    if (request?.coursebag !== undefined) {
```

Comment:
I assume we can do the same thing here?

```suggestion
    if (request?.coursebag) {
```

Reply:
Yeah